### DISCUSSION TITLE
GLB to USDZ Auto-Conversion Quality Settings?

### DISCUSSION THREAD

**Discussion by planetMatrix** [google/model-viewer#4111] [Q&A]
Is there any parameter we can use to control the quality of conversion?

I tried using https://threejs.org/editor/ for manual conversion and ios-src to display. With this the texture quality is at par with GLB, but the USDZ file size increases dramatically.

With Auto-conversion, texture quality is not good and if my understanding is right MV uses Three,js USDZexporter, the same which the editor use.

So how can I control the texture quality output in Auto-Conversion

Many Thanks in advance

**Comment by elalish** [comment]
Indeed, those use the same code as far as I'm aware, so I'm surprised you're seeing different results. Can you upload the two USDZs so we can see the difference? @mrdoob, do you know of anything special your editor does for USDZ conversion?

**Reply by mrdoob** [reply]
It's not doing anything special:

https://github.com/mrdoob/three.js/blob/dev/editor/js/Menubar.File.js#L389-L393
